tests/framework/microvm.py

```diff
@@ -205,6 +208,37 @@ def kill(self):
             self.expect_kill_by_signal = True
             utils.run_cmd("kill -9 {} || true".format(self.screen_pid))
 
+        if self.time_api_requests:
+            api_request_regex = re.compile(
+                r":fc_api\] The API server received a (?P<method>\w+) request on \"(?P<url>(/\w*)+)\" with body (?P<body>.*)\."
```

Comment:
LGTM, should we move it into its own function?

Reply:
Done :)